syntax = "proto3";

package auth;


message Auth{
    string access_token = 1;
    string token_type = 2;
};


message LoginRequest{
    string username = 1;
    string password = 2;
};

message LoginResponse{
    string access_token = 1;
    string token_type = 2;
}


service AuthService{
    rpc Login(LoginRequest) returns (LoginResponse);
}

